assets/css/sensei-course-theme/blockbase-ponyfill.scss

```diff
@@ -203,7 +203,7 @@ a:hover, a:focus {
 }
 
 a:not(.ab-item):not(.screen-reader-shortcut):active, a:not(.ab-item):not(.screen-reader-shortcut):focus {
```

Comment:
Small thing, using [:focus-visible](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) here would be a better fit, it usually won't show the focus ring on mouse click.

Reply:
Thanks @yscik! I haven't used `:focus-visible` before. Will keep in mind.

For this case, I was a bit reluctant in changing any rules, given that this is a full fledged blockbase style lib.

Reply:
I already modified some rules in there, and it was a compiled version to start with for simplicity, so I doubt we'll want to keep it in sync, we can just think of it as a base CSS for content (maybe even rename it).